cocotb/share/lib/vpi/VpiImpl.h

```diff
@@ -66,8 +66,8 @@ static inline int __check_vpi_error(const char *file, const char *func, long lin
     }
 
     gpi_log("cocotb.gpi", loglevel, file, func, line,
-            "VPI Error %s\nPROD %s\nCODE %s\nFILE %s",
-            info.message, info.product, info.code, info.file);
+            "VPI Error %s\nPROD %s\nCODE %s\nFILE %s:%d",
```

Comment:
When I saw such messages in the past, I had assumed they were produced by the simulator by the way they looked.
Is that strange format some kind of standard, or could we make it nicer?

Reply:
This patch adds the line number to the VPI error info from the simulator. This was very helpful to track down exactly where the error was happening in Verilator while debugging VPI issues. 
The error message is printing the relevant members of the VPI error info struct:
https://github.com/cocotb/cocotb/blob/97e28803308d09d2bd885e4279d5036ef9861d42/cocotb/share/include/vpi_user.h#L767
The VPI error message above is nested in the GPI error message, which will show file, function and line number of where in the GPI layer an error occurred.  
Do you have any suggestions to improve this?

Reply:
I understand what it says, it's just that the four letter all-uppercase "identifiers" separated by newlines look like they were formatted for machine readability instead of being for humans.
PROD we don't need at all IMO, the newlines can go, and the rest should read like a sentence. (I'm typing this on a phone, I can try to be more specific later).

Reply:
You could perhaps use two separate log lines, and pass a custom file/func/line into the second one

Reply:
OK, I've updated to separate 2 log lines as suggested. 
Here's an example error log message:
```
     0.00ns ERROR    cocotb.gpi                                 VpiCbHdl.cpp:172  in initialise                      VPI error
     0.00ns ERROR    cocotb.gpi                            verilated_vpi.cpp:1615 in Verilator                       vpi_get_value: Unsupported vpiHandle (0x0)
```

Reply:
Thanks, that looks much nicer in my opinion.